### PR TITLE
Add Common class library

### PR DESCRIPTION
**Pull request by TiagoGraf89**
Move swagger to common

### CODE REVIEW

File: Apps/Immunization/src/Immunization.csproj [hunk 1]
```diff
@@ -35,4 +35,7 @@
     <PackageReference Include="Swashbuckle.AspNetCore" Version="4.0.1" />    
     <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\Common\Common.csproj" />
```
Comment (FuriousLlama) [inline Apps/Immunization/src/Immunization.csproj]: This looks awkward

**Comment by sslaws** [comment]
SonarQube Report: https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_92

**Comment by sslaws** [comment]
SonarQube Report: https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_92

**Comment by sslaws** [comment]
SonarQube Report: https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_92

**Comment by healthgatewaybc** [comment]
SonarQube Report: https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=webclient_pr_92

**Comment by sslaws** [comment]
SonarQube Report: https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_92

**Comment by sslaws** [comment]
SonarQube Report: https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_92

**Comment by sslaws** [comment]
SonarQube Report: https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_92

**Comment by healthgatewaybc** [comment]
SonarQube Report: https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=webclient_pr_92

**Pull request merged**